tasklib/backends.py

```diff
@@ -427,3 +427,34 @@ def valid(output):
 
     def sync(self):
         self.execute_command(['sync'])
+
+    def _get_task_attrs(self):
+        TASK_STANDARD_ATTRS = [
```

Comment:
I think this would be good to define on the class itself (either the backend or `Task`) rather than in the method.

Reply:
Moved on commit 7805380